autoware_adapi_v1_msgs/perception/msg/PredictedObjectKinematics.msg

```diff
@@ -0,0 +1,5 @@
+geometry_msgs/PoseWithCovariance initial_pose_with_covariance
+geometry_msgs/TwistWithCovariance initial_twist_with_covariance
+geometry_msgs/AccelWithCovariance initial_acceleration_with_covariance
+
+autoware_adapi_v1_msgs/PredictedPath[] predicted_paths
```

Comment:
I don't think `predicted_paths` will be used much. How about removing it for now and adding it if needed?